src/store/migrations.test.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+import { migrationFromZeroToOne } from './migrations';
+
+const ENGINE_KEY = 'test-key';
+
+describe('Redux migrations', () => {
+  describe('Version 0 -> Version 1', () => {
+    it('does nothing with no initial state to work with', () => {
+      const persistedState = null;
+
+      const expectedOutput = persistedState;
+      const actualOutput = migrationFromZeroToOne(persistedState);
+
+      expect(actualOutput).toEqual(expectedOutput);
+    });
+
+    it('does not affect compatible state', () => {
+      const persistedState = {
+        projects: {
+          byId: {
+            hello: {},
+            goodbye: {},
+          },
+        },
+      };
+
+      const expectedOutput = persistedState;
+      const actualOutput = migrationFromZeroToOne(persistedState);
+
+      expect(actualOutput).toEqual(expectedOutput);
+    });
+
+    it('handles stringified state, and deletes tasks', () => {
+      const persistedState = JSON.stringify({
+        projects: {
+          byId: {
+            hello: {},
+            goodbye: {},
+          },
+        },
+        tasks: {
+          'hello-start': {},
+          'hello-build': {},
+          'goodbye-start': {},
+          'goodbye-build': {},
+        },
+      });
+
+      const expectedOutput = {
+        projects: {
+          byId: {
+            hello: {},
+            goodbye: {},
+          },
+        },
+      };
+      const actualOutput = migrationFromZeroToOne(persistedState);
+
+      expect(actualOutput).toEqual(expectedOutput);
+    });
+  });
+});
```

Comment:
@mathieudutour I know the docs on your migration library provide [different test instructions](https://github.com/mathieudutour/redux-storage-decorator-migrate#testing-migrations-without-a-store-applying-against-ad-hoc-state), but I feel like testing the transform function is sufficient; I can trust that your library does what it says.

Let me know if there's a disadvantage I'm not seeing to this approach though!